lahja/base.py

```diff
@@ -232,8 +232,12 @@ def broadcast_nowait(
     @abstractmethod
     async def wait_until_remote_subscriptions_change(self) -> None:
         """
+<<<<<<< HEAD
         Block until any subscription change occurs on any remote endpoint or
         the set of remote endpoints changes
+=======
+        Block until any subscription change occurs on any remote endpoint.
+>>>>>>> Generic API for waiting for subscription and connection changes
```

Comment:
You probably didn't intend to commit this :sweat_smile:

Reply:
It is removed in #88